lib/realtime_signs_web/controllers/monitoring_controller.ex

```diff
@@ -39,4 +40,17 @@ defmodule RealtimeSignsWeb.MonitoringController do
 
     send_resp(conn, 200, "")
   end
+
+  def update_active_headend(conn, %{"ip" => active_ip} = _params) do
```

Comment:
Are there any security considerations with exposing this endpoint? I assume we're primarily relying on the network topology to restrict access, but since this impacts the running system, I wanted to check.

Reply:
Yeah, for now it's just the network topology that we're relying on.

Personally, I think access to the url is adequately restricted at the moment. If anything, since we're moving this value to S3, the value is arguably more openly and easily changeable via a direct upload to the bucket. I think adding at least a basic security layer for this endpoint is a good idea, but not sure how much we'd necessarily gain from that. Maybe we should add a separate task for setting up some authentication for the phoenix app as a whole though.